src/js/components/form-elements/TableBodyVirtualized.jsx

```diff
@@ -20,24 +20,24 @@ class TableBodyVirtualized extends Component {
     const { fieldsConfig: { subfieldKey, getDynamicRowAttr }, fields, properties } = this.props;
 
     if (!subfieldKey) {
-      return 45;
+      return 50;
```

Comment:
Seems like a good candidate for adding a constant / global config property :)

Reply:
it's used only in this one method